etl/pipelines/gdl/loaders/load_all_files.py

```diff
@@ -0,0 +1,56 @@
+"""
+Load GDL annual data from JSON
+"""
+
+from load_iso_countries import load_iso_countries
+from load_gdl_regions import load_gdl_regions
+from load_gdl_national import load_gdl_national
+from load_gdl_subnational import load_gdl_subnational
+from load_annual_data import load_all_tables
+from loader_utils import init_db_session
+from models import (
+    GdlAnnual,
+    GdlNational,
+    GdlSubnational,
+    GdlRegion,
+    IsoCountry,
+)
+
+# Data paths
+ISO_META_PATH = "../json/iso_countries.json"
+REGIONS_META_PATH = "../json/gdl_regions.json"
+NATIONAL_GEOJSON_PATH = "../geojson/GDL_V6.3_national_small.json"
+SUBNATIONAL_GEOJSON_PATH = "../geojson/gdl_v6.3_large_visvaligram_weighted_0.02.json"
+DEVELOPMENT_ANNUAL_PATH = "../json/development.json"
+EDUCATION_ANNUAL_PATH = "../json/education.json"
+HEALTHCARE_ANNUAL_PATH = "../json/healthcare.json"
+INCOME_ANNUAL_PATH = "../json/income.json"
```

Comment:
Avoid committing all the JSON and GeoJSON - this should all be downloaded and transformed, direct from the [GDL site](https://globaldatalab.org/shdi/) if that's reasonable, or packed in a Zenodo dataset as a redistribution (their [terms of use](https://globaldatalab.org/termsofuse/) explicitly allow sharing and adaptation with clear citation and acknowledgment) if that's an easier starting point.

Can you also double check the indicators are from the latest version of the SHDI? Version 8.1 was released on 23 October 2024.